yacc_error1.py returned with 1

Traceback (most recent call last):
  File "/home/herczy/devel/ply/test/yacc_error1.py", line 64, in <module>
    yacc.yacc()
  File "/home/herczy/devel/ply/ply/yacc.py", line 3157, in yacc
    raise YaccError("Unable to build parser")
ply.yacc.YaccError: Unable to build parser

ERROR: /home/herczy/devel/ply/test/yacc_error1.py:61: p_error() requires 1 argument
